### PR TITLE
feat: add `FileSlice` support for `html/text` mime type

### PR DESCRIPTION
**Pull request by baudoliver7**
We added support for `html/text` and introduced some unit and IT tests.

Fix: #94

### CODE REVIEW

**Comment by baudoliver7** [comment]
@artipie/maintainers ready to be reviewed :)

File: src/main/java/com/artipie/files/BlobListFormat.java [hunk 1]
```diff
@@ -59,6 +59,31 @@
...
+        ),
+
+        /**
+         * HTML format renders keys as simple markdown with ul, li and a tags.
+         */
+        HTML(
+            keys -> String.format(
+                String.join(
+                    "\n",
+                    "<html>",
```
Comment (g4s8) [inline src/main/java/com/artipie/files/BlobListFormat.java:71]: Please add `<!DOCTYPE html>` and
```html
<head><meta charset="utf-8"/></head>
```

**Review by g4s8 — CHANGES_REQUESTED** [review]
@baudoliver7 thanks, just small improvement

File: src/main/java/com/artipie/files/BlobListFormat.java (hunk 1 above)
Comment (baudoliver7) [reply src/main/java/com/artipie/files/BlobListFormat.java:72]: @g4s8 fixed.

**Review by g4s8 — APPROVED** [review]
@baudoliver7 thanks

**Pull request merged**